### DISCUSSION TITLE
Notifications via keyboard backlight

### DISCUSSION THREAD

**Discussion by prmx** [seerge/g-helper#1156] [Ideas]
Hi there! 

I know this idea may be a bit out of scope for G-Helper, but it could be a great productivity feature.

Idea: when a new user notification arrives, the keyboard backlight temporarily switches from current mode to notification mode. This will be a visual indicator for the user.

For example, your laptop has the following keyboard settings: 0% brightness, Static mode. On a new notification from Outlook, it temporarily switches to 25% brightness and Breathe mode to indicate that the user has received an email.

In the straightforward way, this feature can respond to all user notifications, in a more advanced way, the reaction can be filtered by application name.

Thank you for this amazing tool!

**Comment by seerge** [comment]
@prmx hello :) I'm afraid this is a very specific and narrow scenario. Also app doesn't track windows notifications anyhow in a first place.

You can try to write some script by yourself, if it's important to you, setting keyboard color is literally 2 USB packets.
